### ISSUE TITLE
be able to filter 'enabled', 'disabled', 'all' encounters

### ISSUE THREAD

**Issue by mwiegant** [mwiegant/avarrocka#6]
Depends on Issue #5.

Add a filter button (using angular, this can be done) that will allow the filtering of encounters by 'enabled' or 'disabled'. I think it should be possible to have no filtering too, so that all encounters show up.

When filtering by 'enabled' or 'disabled', only show the encounters that match that filter.

I.e., if filtering by 'enabled', only show those encounters that are 'enabled'.

_This issue is more of a nice-to-have than a need-to-have_
